js/layout-javascript/agenda-code.js

```diff
@@ -457,7 +486,11 @@ DynamicList.prototype.attachObservers = function() {
         allowClick = true;
       }, 100);
     })
-    .on('click', '.agenda-list-item', function(event) {
+    .on('click keydown', '.agenda-list-item', function(event) {
+      if (event.type === 'keydown' && event.keyCode !== 32 && event.keyCode !== 13) {
```

Comment:
There is more in this PR that still needs replacing.

```suggestion
      if (!_this.Utils.Event.isExecute(event)) {
```